store/store.go

```diff
@@ -192,8 +192,12 @@ func getStructFields(s interface{}) []string {
 	return fields
 }
 
+// Deltas enabled by default on classic, but allow opting in or out on both classic and core.
 func useDeltas() bool {
-	return osutil.GetenvBool("SNAPD_USE_DELTAS_EXPERIMENTAL")
+	if u := os.Getenv("SNAPD_USE_DELTAS_EXPERIMENTAL"); u != "" {
+		return osutil.GetenvBool("SNAPD_USE_DELTAS_EXPERIMENTAL")
```

Comment:
A shorter version of this function could be: 
`return osutil.GetenvBool("SNAPD_USE_DELTAS_EXPERIMENTAL", release.OnClassic)`
or maybe:
```
default := release.OnClassic
return  osutil.GetenvBool("SNAPD_USE_DELTAS_EXPERIMENTAL", default)
```
to make it more obvious.